# This message contains the output from the actuator interface node

# motor commands (per wheel)
std_msgs/Float64MultiArray omega_arr # DC motor cmds
std_msgs/Float64MultiArray phi_arr # Stepper motor cmds